src/phoenix/version.py

```diff
@@ -1 +1 @@
-__version__ = "4.5.0"
+__version__ = "4.5.1"
```

Comment:
```suggestion
__version__ = "4.5.0"
```
We have automation for versions so no need to bump here:)